### PR TITLE
added wallet addresses to read balance

### PR DESCRIPTION
**Pull request by WlinkNET**

### CODE REVIEW

File: app/routes/network.tsx [hunk 1]
```diff
@@ -8,8 +8,20 @@ async function loadNetworkState() {
...
-  const circulatingSupply = totalIssuanceNumber - budget;
+  const addresses = [
+    "d1CNDotJXNPvnSA5EQXpSbkUyXBVmaggkARY7kcgXim4BqeBJ",
+    "d1GZ8GxP3KzKJGRYmp9HMwxurnSKx3ACcqeZqLY5kpbLEyjzE",
+    "d1GA9xWx3WgpQHp8LHCXHbYoZdvjY3NHhU6gR2fsdVCiC4TdF",
+    "d1ESJKwsk6zP8tBNJABUnf8mtKcqo1U2UVG7iEZ7uytGbWKAL",
+    "d1EVSxVDFMMDa79NzV2EvW66PpdD1uLW9aQXjhWZefUfp8Mhf",
+  ];
+  let balanceSum = 0;
+  for (const address of addresses) {
```
Comment (fedosov) [inline app/routes/network.tsx:19]: Let's do this in parallel. Check out the `await Promise.all` method. This should be faster that doing this in sequence.

File: app/routes/network.tsx [hunk 2]
```diff
@@ -8,8 +8,20 @@ async function loadNetworkState() {
...
+    const accountInfo = await api.query.system.account(address);
+    const balance = Number(accountInfo.data.free.toString());
```
Comment (fedosov) [inline app/routes/network.tsx:21]: Can we use `toBigInt` here and avoid string/number conversion for simplicity?

File: app/routes/network.tsx (hunk 1 above)
Comment (WlinkNET) [reply app/routes/network.tsx:19]: Ok will check it

File: app/routes/network.tsx (hunk 2 above)
Comment (WlinkNET) [reply app/routes/network.tsx:21]: Im not sure but I will test it

File: app/routes/network.tsx (hunk 1 above)
Comment (WlinkNET) [reply app/routes/network.tsx:19]: ` const balancePromises = addresses.map(async (address) => {
    const accountInfo = await api.query.system.account(address);
    const balance = Number(accountInfo.data.free.toString());
    return balance;
  });

  const balances = await Promise.all(balancePromises);
  const balanceSum = balances.reduce((acc, val) => acc + val, 0);
  const circulatingSupply = totalIssuanceNumber - balanceSum;`
We can do it like this if is that satisfied?

File: app/routes/network.tsx (hunk 2 above)
Comment (WlinkNET) [reply app/routes/network.tsx:21]: If we want to use BigInt we must refactor all of it we cant mix types

**Pull request merged**